be/src/common/status.h

```diff
@@ -413,6 +413,17 @@ struct StatusInstance {
         }                                     \
     } while (false)
 
+#define CHECK_RETURN_AUX(fmt, ...) fmt " " #__VA_ARGS__
+
+#define CHECK_RETURN(condition, ...)                                                  \
```

Comment:
This is not a good name. What do you want is `RETURN_ERROR_IF_FALSE`?
But I don't think it is a good idea to add this macro.
It is better to handle unexpected conditions specifically.

Reply:
Yes, in many conditions, we need to return InternalError so we can handle it. This macro is used for these cases.